docs/core/extensions/httpclient-factory.md

```diff
@@ -293,6 +293,77 @@ A further workaround can follow with an extension method for registering a scope
 
 For more information, see the [full example](https://github.com/dotnet/docs/tree/main/docs/core/extensions/snippets/http/scopeworkaround).
 
+## Avoid depending on "factory-default" Primary Handler
+
+In this section, we use the term _"factory-default" Primary Handler_, by which we mean the Primary Handler that the default `IHttpClientFactory` implementation (or more precisely, the default `HttpMessageHandlerBuilder` implementation) assigns if _not configured in any way_ whatsoever.
```

Comment:
```suggestion
In this section, the term _"factory-default" Primary Handler_ refers to the Primary Handler that the default `IHttpClientFactory` implementation (or more precisely, the default `HttpMessageHandlerBuilder` implementation) assigns if _not configured in any way_ whatsoever.
```

Reply:
Style rules want us to avoid we/our in docs - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we?toc=%2Fwriting-style-guide-msft-internal%2Ftoc.json&bc=%2Fwriting-style-guide-msft-internal%2Fbreadcrumb%2Ftoc.json